src/igetntbi.f

```diff
@@ -1,27 +1,26 @@
 C> @file
-C> @author ATOR @date 2009-03-23
-	
-C> THIS FUNCTION RETURNS THE NEXT AVAILABLE INDEX FOR
-C>   STORING AN ENTRY WITHIN INTERNAL BUFR TABLE CTB.
-C>
-C> PROGRAM HISTORY LOG:
-C> 2009-03-23  J. ATOR    -- ORIGINAL AUTHOR
-C> 2014-12-10  J. ATOR    -- USE MODULES INSTEAD OF COMMON BLOCKS
+C> @brief Get the next index for storing an entry within an internal
+C> DX BUFR table
+
+C> This function returns the next available index for storing an
+C> entry within a specified internal DX BUFR table.
 C>
-C> USAGE:    CALL IGETNTBI ( LUN, CTB )
-C>   INPUT ARGUMENT LIST:
-C>     LUN      - INTEGER: I/O STREAM INDEX INTO INTERNAL MEMORY ARRAYS
-C>     CTB      - CHARACTER*1: INTERNAL BUFR TABLE FROM WHICH TO RETURN
-C>                THE NEXT AVAILABLE INDEX ('A','B', OR 'D')
+C> @author J. Ator
+C> @date 2009-03-23
 C>
-C>   OUTPUT ARGUMENT LIST:
-C>     IGETNTBI  - INTEGER: NEXT AVAILABLE INDEX IN TABLE CTB
+C> @param[in]  LUN    - integer: Internal I/O stream index associated
+C>                      with BUFR file
+C> @param[in]  CTB    - character: Type of internal DX BUFR table for
```

Comment:
Actually, I've found that the type only shows up automatically in Doxygen if the variable is explicitly declared.  However, a lot of variables in this and other legacy NCEP libraries use implicit Fortran typing, e.g. any variable name starting with the letters I-N is implicitly an integer, rather than being explicitly declared.